recipes/tllist/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.44.0"
```

Comment:
Currently, we use `1.43.1`, so this PR won´t succeed until we change the version (usually not before one month after the release)

Reply:
Ok I corrected this. However the instructions of the PR process explicitly requires and points to the latest release (ie 1.44.0). Maybe it shall be updated to add the one month period requirement ?